index.js

```diff
@@ -5,38 +5,48 @@ const request = require('request');
 const rp = require('request-promise-native');
 const wallpaper = require('wallpaper');
 
-const change = () => {
-    const url = `https://www.reddit.com/r/${randomElement(config.subreddits)}/${config.sort}.json?t=${config.from}limit=${config.limit}`;
-    const options = {
-        uri: url,
-        json: true,
-        simple: true,
-    };
+const maxRetry = config.maxRetry
 
-    const path = config.directory + '/wallpaper';
-    let name = '';
-
-    if (!fs.existsSync(config.directory)) {
-        fs.mkdirSync(config.directory);
+const change = (start) => {
```

Comment:
Give the start parameter an initial value of 0. [Check out this MDN article](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters).